src/main/kotlin/inc/evil/d17map/services/EquipmentService.kt

```diff
@@ -28,4 +28,15 @@ class EquipmentService (val equipmentRepository : EquipmentRepository) {
             equipmentRepository.delete(it)
         } ?: throw EntityNotFoundException("Equipment not found with id: $equipmentId")
     }
+
+    fun getAll(): List<EquipmentResponse> {
+        val equipments = equipmentRepository.findAll()
+
+        return equipments.map { equipment ->
+            EquipmentResponse(
```

Comment:
Wydaje mi się, że jest to ten sam case co w pliku `ClassroomMapper`  - jeśli faktycznie tak jest, to zunifikujmy.
```
 return ClassroomResponse(
            id = classroom.id!!,                            <----
            name = classroom.name,
            description = classroom.description,
            capacity = classroom.capacity,
            equipments = equipmentResponses
        ) 
```